code/mob/living/critter/small_animal.dm

```diff
@@ -2584,6 +2584,15 @@ var/list/mob_bird_species = list("smallowl" = /mob/living/critter/small_animal/b
 				return 2
 		return ..()
 
+	specific_emotes(var/act, var/param = null, var/voluntary = 0)
+		switch (act)
+			if ("dance")
+				if (src.emote_check(voluntary, 50))
+					SPAWN_DBG(0)
```

Comment:
```suggestion
```
This is indented wrong and fortunately isn't necessary since `animate` doesn't block, it just queues stuff up for the client.